src/RA_Defs.h

```diff
@@ -1,3 +1,5 @@
+#ifndef RA_DEFS_H
+#define RA_DEFS_H
```

Comment:
isn't this redundant with the `#pragma once` on the next line?

Reply:
I was taught to always use include guards. It might seem redundant, it can go away.